usr/share/rear/conf/default.conf

```diff
@@ -2169,6 +2168,23 @@ COPY_AS_IS_NBKDC=()
 COPY_AS_IS_EXCLUDE_NBKDC=()
 ####
 
+
+####
+# BACKUP=NFS_SERVER
+##
+# Note: This is for NFS_Server
+#
+REQUIRED_PROGS_NFS_SERVER=("rpc.nfsd" "rpcbind" "rpc.statd" "rpc.nfsd" "rpc.mountd" "exportfs" "rpcinfo" "rpcdebug" "ss")
+PROGS_NFS_SERVER=("rpcdebug" "rpcinfo")
+REQUIRED_PROGS_NFS_SERVER_V3=("rpc.statd" "rpc.idmapd" "rpc.gssd")
+NFS_SERVER_EXPORT_OPTS="rw,no_root_squash,async,no_subtree_check"
+NFS_SERVER_TRUSTED="*"
+NFS_SERVER_MODULES=("nfs" "nfsd")
```

Comment:
You'll have to check also if the modules are actually available

Reply:
Main thought is that if `rear mkrescue` works, then there is a high chance that the configured features will also work. At the moment this is more on the level of "I hope that it will work"

Reply:
What is the best method to check if the modules are available?

Reply:
Regarding a check if kernel modules are available:
I assume what is meant is "available as files"
(i.e. no builtin kernel "modules").
You may have a look at the code and the comments of the
```
function modinfo_filename ...
```
in usr/share/rear/build/GNU/Linux/400_copy_modules.sh

In that script at its end we add all what there is
in MODULES_LOAD to "$ROOTFS_DIR/etc/modules" which is used in
skel/default/etc/scripts/system-setup.d/40-start-udev-or-load-modules.sh
that gets installed in the ReaR recovery system as
/etc/scripts/system-setup.d/40-start-udev-or-load-modules.sh
which basically calls `modprobe -v $module` for all
what there is in MODULES_LOAD so
```
MODULES_LOAD+=( QQQ )
```
results that `modprobe -v QQQ` will be called during
ReaR recovery system startupt which does not cause harm
but only an error message
```
# modprobe -v QQQ
modprobe: FATAL: Module QQQ not found in directory /lib/modules/...
```
which seems to be OK from my current point of view.